src/App.js

```diff
@@ -1,12 +1,12 @@
-import { useState } from 'react';
+import React from 'react';
 import './App.css';
 import Header from './components/header/Header';
 import AddTodo from './components/AddTodo'
 
 
 function App() {
   
-  const [todos, setTodos] = useState([]);
+  
```

Comment:
Put this state variable back.

Pass it to the component that needs it via props